config/install/field.field.localgov_alert_banner.localgov_alert_banner.field_localgov_link.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.localgov_alert_banner.field_localgov_link
+    - localgov_alert_banner.localgov_alert_banner_type.localgov_alert_banner
+  module:
+    - link
+id: localgov_alert_banner.localgov_alert_banner.field_localgov_link
+field_name: field_localgov_link
```

Comment:
https://github.com/localgovdrupal/localgov_services/blob/b8d65a1a361a05ec368f3156f7d818c12bc93fff/modules/localgov_services_page/config/install/field.field.node.localgov_services_page.localgov_services_parent.yml#L12

The confusing might arise from the fact that most of the fields haven't been touched and are still `field_something`. But here it's keeping with the `localgov_`